src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -89,14 +89,14 @@ export class AutoColumnSize extends BasePlugin {
       cachedColumnHeaders: [],
     });
     /**
-     * Instance of {@link GhostTable} for rows and columns size calculations.
+     * Instance of {@link ghost-table GhostTable} for rows and columns size calculations.
```

Comment:
I guess, that API reference isn't generated for GhostTable. Consider about what with this fact.

Reply:
it is marked also as private, should we generate API Ref for that? (are we generating it now?)

Reply:
Just make it 1:1, removing it would require rethinking redirects and other aspects. Documentation review is not a part of this scope

Reply:
That is a private field that does not be published on the documentation. I revert changes